### PR TITLE
Merged Status and Lobby tab for new_player

### PR DESCRIPTION
**Pull request by DamianX**

### CODE REVIEW

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -372,7 +370,7 @@
 				if(istype(P.cartridge,/obj/item/weapon/cartridge/trader))
 					var/mob/living/L = get_holder_of_type(P,/mob/living)
 					if(L)
-						L.show_message("[bicon(P)] <b>Message from U���8�E1��Ћ (T�u1B��), </b>\"Caw. Cousin [character] detected in sector.\".", 2)
+						L.show_message("[bicon(P)] <b>Message from U¦ŸÉ8¥E1ÀÓÐ‹ (T¥u1B¤Õ), </b>\"Caw. Cousin [character] detected in sector.\".", 2)
```
Comment (Exxion) [inline code/modules/mob/new_player/new_player.dm]: Look out

Comment (DamianX) [reply code/modules/mob/new_player/new_player.dm]: Apparently it was using the wrong encoding and turning it to UTF-8 did that. It's garbage text anyway, I don't think this matters.

Comment (Kurfursten) [reply code/modules/mob/new_player/new_player.dm]: That's correct. The way I got this text was by force opening a PDF document with notepad and pasting the scrambled characters.

**Comment by DamianX** [comment]
Rebased to fix conflicts, secret repo PR is ready as well.

**Pull request merged**